xml/smtp_banners.xml

```diff
@@ -1690,20 +1871,21 @@ The system or service fingerprint with the highest certainty overwrites the othe
     <param pos="3" name="service.version.version"/>
     <param pos="4" name="system.time"/>
   </fingerprint>
-  <fingerprint pattern="^(\S+) E?SMTP Perl" flags="REG_ICASE">
+  <fingerprint pattern="^(?i)(\S+) E?SMTP Perl">
```

Comment:
`(?i)(\S+)` is a little odd.  Just make it `(\S+).

Reply:
I can't find this banner in our dataset so I set the whole regex to be case insensitive.